scripts/dev_setup.sh

```diff
@@ -23,7 +23,7 @@ VAULT_VERSION=1.5.0
 Z3_VERSION=4.11.0
```

Comment:
Z3 also recently has a newer version: https://github.com/Z3Prover/z3/releases/tag/z3-4.11.2

The Z3 version seems unrelated to the Prover issue because upgrading Boogie version itself fixes the Prover issue. However, it may be good to upgrade the Z3 version as well.

Reply:
I do not went to set a new z3 version before we know this actually fixes it. We have no idea what the issue is, and whether it is in boogie or Z3.